src/quickstarts/azure-openai/extensions-ai/02-build-chat-app/ChatAppAI.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
I think we should stay on .NET 8 for now, to make sure we can catch the broadest set of customers.